frontend/src/translatePlugin.js

```diff
@@ -0,0 +1,83 @@
+class TranslationsHandler {
+	constructor() {
+		this._messages = {};
+
+		const load = async () => {
+			if (window.frappe?.boot?.messages) {
+				this._messages = window.frappe.boot.messages;
```

Comment:
Should we go the _"transitional"_ way by adding messages in `get_boot` in hrms/www/hrms.py, or do we wait for the eventual merging of the sister PR https://github.com/frappe/frappe/pull/24435?

Reply:
Messages are loaded in bootinfo for the time being, will add only_hash=True param once merged